ONSdigital/eq-questionnaire-validator
# Replace BEIS with DBT-DSIT theme

### What is the context of this PR?
Removes the BEIS theme and replaces it with the DBT-DSIT theme

### How to review
- Check changes make sense
- Check theme names match ones in runner PR

### Checklist
